internal/cmd/cmd.go

```diff
@@ -23,7 +23,7 @@ func Make() *cobra.Command {
 	app.AddCommand(
 		makeLoginCmd(),
 		makeLogoutCmd(),
-		makeShellCmd(),
+		shCmd(),
```

Comment:
Why the rename of this particular command? Consistency seems better here, so we should change them all or not change this one.

Reply:
Yeah fair point. Just updated all uses of `make*`